src/lake/Lake/Config/Package.lean

```diff
@@ -322,6 +335,10 @@ namespace Package
 @[inline] def backend (self : Package) : Backend :=
   self.config.backend
 
+/-- The package's `leanOptions` configuration. -/
+@[inline] def leanOptions (self : Package) : Array LeanOption :=
+  self.config.leanOptions
```

Comment:
As noted above, these should be combined with package's other options in the package definitions themselves (which will require reordering these definitions).